src/components/app/component.tsx

```diff
@@ -1,5 +1,9 @@
 import * as React from "react";
+import MatchingCockpit from "./matching/MatchingCockpit";
 
 export const AppComponent = () => (
-  <h1>AccommoDate</h1>
+  <div>
+    <h1>AccommoDate</h1>
+    <MatchingCockpit/>
```

Comment:
we'll need react router, not your fault though

Reply:
does it require any changes from my side?